KsApi/models/graphql/adapters/Project+FetchProjectQueryData.swift

```diff
@@ -54,6 +55,9 @@ extension Project {
         Reward.reward(from: fragment)
       } ?? []
 
+    let emptyRewards = [noRewardReward(from: data.project?.fragments.projectFragment)]
+    let updatedRewardsWithNoReward = emptyRewards + rewards
```

Comment:
I wonder if a better name for this constant would be `allRewards` or something similar.